curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8b9770050d217d2247801.md

```diff
@@ -149,7 +149,7 @@ button3.onclick = fightDragon;
 
 --fcc-editable-region--
 function update(location) {
-  button1.innerText = location["button text"];
+  button1.innerText = location["button text"][0];
```

Comment:
These changes are incorrect.

Please revert back to the original.

The issue wants you to update the description text not the seed code here.

Please make the changes outlined in the issue here

https://github.com/freeCodeCamp/freeCodeCamp/issues/53426

Reply:
Changed the description now